OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -423,6 +422,9 @@ - (OPTLYVariation *)sendImpressionEventFor:(OPTLYExperiment *)experiment
                                                  callback(error);
                                              }
                                          }];
+    NSString *_userId = userId ? userId : @"";
+    NSDictionary *_attributes = attributes ? attributes : [NSDictionary new];
+    [_notificationCenter sendNotifications:OPTLYNotificationTypeActivate args:experiment, _userId, _attributes, variation, impressionEventParams, nil];
     return variation;
```

Comment:
Same here.
In order to implement variable arguments in sendNotifications method, I make sure no argument could be pass as nil which can break the argument list.